### PR TITLE
config options, test framework, dev env, and polished printing

### PR DESCRIPTION
**Pull request by KerstenBreuer**
- configuration can now be performed by:
  - env variables
  - config file
  - CLI parameters
- added a testing framework and basic tests for existing functions
- set up a vscode dev container environment
- Exception are now not handled anymore (user sees the Traceback)
- polished printing
- functions return response when being used programmatically

### CODE REVIEW

File: datameta_client.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Alternative console entry point.
+Usage: `python3 ./datameta_client.py --help`
+"""
+
+from datameta_client.__main__ import app
+
+app()
```
Comment (lkuchenb) [inline datameta_client.py:23]: What is this file needed for?

**Review by lkuchenb — APPROVED** [review]
Thanks a lot, looks very solid to me 👍 

Let's move this forward now, but let's get rid of the backend related tests here in a followup (#9).

File: datameta_client.py (hunk 1 above)
Comment (KerstenBreuer) [reply datameta_client.py:23]: It's common for python packages to have this alernative console entry point. Without installation you can use: `python3 ./datameta_client.py --help`
(assuming all deps are there)
